drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenterTest.java

```diff
@@ -620,11 +626,14 @@ private void linkTables() {
         final GuidedDecisionTableView dtView3 = mock(GuidedDecisionTableView.class);
         final GuidedDecisionTableView.Presenter dtPresenter2 = setupPresenter();
         final GuidedDecisionTableView.Presenter dtPresenter3 = setupPresenter();
-        final Set<GuidedDecisionTableView.Presenter> dtPresenters = new HashSet<GuidedDecisionTableView.Presenter>() {{
-            add(dtPresenter);
-            add(dtPresenter2);
-            add(dtPresenter3);
-        }};
+        final Set<GuidedDecisionTableView.Presenter> dtPresenters = new HashSet<GuidedDecisionTableView.Presenter>() {
+
```

Comment:
Is this Eclipse's formatting of static initialiser blocks (in anonymous sub-classes!)